src/services/git.js

```diff
@@ -21,20 +19,20 @@ class GitService {
   }
 
   /**
-   * Create and check if command is valid
-   * @param action
-   * @param id
-   * @param key
-   * @param name
-   * @param payload
-   * @param callback
-   */
+     * Create and check if command is valid
+     * @param action
+     * @param id
+     * @param key
+     * @param name
+     * @param payload
+     * @param callback
+     */
   command (args, callback) {
-    const options = {cwd: this.currentSharedFolder, shell: true}
+    validateCommand(args.script, gitRegex)
```

Comment:
in case `validateCommand` fails, the `callback` still need to be called with an error.